.github/workflows/automatic-doc-checks.yml

```
@@ -1,16 +1,22 @@
-name: Main Documentation Checks
+#
+name: Automatic doc checks
 
 on:
-  - push
-  - pull_request
-  - workflow_dispatch
+  push:
+    branches: [ main ]
+  pull_request:
+
+  workflow_dispatch:
+    # Manual trigger
```

You can consolidate this section:

```suggestion
  push:
    branches: [ main ]
  pull_request:
  workflow_dispatch:
```

---

Since this is a file directly from the starter pack, I'm going to leave as is for the time being.